pkg/bpf/bpf.go

```diff
@@ -264,6 +273,87 @@ func recoverVersionMap(pinPath string) *ebpf.Map {
 	return versionMap
 }
 
+func (l *BpfLoader) skipSpecificProbe() {
```

Comment:
we can define an struct to hold all the bpf options, then call this function `UpdateBpfProgOptions`